### PR TITLE
Feat/redux book

### LINKED ISSUE #4: As a user…. I should be able to collect and favorite cat facts to my Cat Bible

### LINKED ISSUE #6: Create Bible Component
Skeleton HTML and CSS component with spots where data will be plugged in later.

### PR DESCRIPTION
**Pull request by jsahim**
Change summary: Renamed Bible to Book component and added Book slice with functionality to save favorited facts to state. Added Router - routes, redirects and switches.
Methods/Functions: addFact() - reducer action

Close #6 Close #4

### CODE REVIEW

File: src/components/App/App.js
Comment (Jesusborjas006) [inline src/components/App/App.js:1]: Nice job adding Router to our application!

File: src/components/Result/Result.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ function Result() {
   const dayNum = useSelector((state) => state.day.value);
   const dispatch = useDispatch();
 
-  const getNextDay = dayNum < 7 ? `/${dayNum}/day` : "/finale";
+  const getNextDay = dayNum < 7 ? `/story/${dayNum}/day` : "/story/finale";
```
Comment (qrispi) [inline src/components/Result/Result.js:11]: Oh ternary... you will forever have a special place in my heart.

File: src/features/bookSlice.js [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { createSlice } from "@reduxjs/toolkit";
+
+export const bookSlice = createSlice({
```
Comment (qrispi) [inline src/features/bookSlice.js:3]: You get a slice, and you get a slice, everybody gets a slice!

**Review by BriBourassa — APPROVED** [review]

File: src/components/Book/Book.js
Comment (Jesusborjas006) [inline src/components/Book/Book.js:1]: Async and await looks very cool!

File: src/features/bookSlice.js
Comment (Jesusborjas006) [inline src/features/bookSlice.js:1]: Nice job adding slice for the book that's storing facts

**Review by Jesusborjas006 — APPROVED** [review]

**Pull request merged**